docs/source/reference/providers.rst

```diff
@@ -97,7 +88,6 @@ Then, we can attach a toy circuit and send it on our session
 
 >>> processor.set_circuit(pcvl.Circuit(m=2, name="a-toy-circuit") // pcvl.BS.H())
 >>> processor.with_input(pcvl.BasicState("|0,1>"))
->>> processor.min_detected_photons_filter(1)
```

Comment:
This line is mandatory to launch any job with a `RemoteProcessor`.
Don't remove it.